synapse/rest/admin/users.py

```diff
@@ -708,3 +719,29 @@ async def on_GET(self, request, user_id):
 
         ret = {"joined_rooms": list(room_ids), "total": len(room_ids)}
         return 200, ret
+
+
+class PushersRestServlet(RestServlet):
+    PATTERNS = admin_patterns("/users/(?P<user_id>[^/]*)/pushers$")
+
+    def __init__(self, hs):
+        self.is_mine = hs.is_mine
+        self.store = hs.get_datastore()
+        self.auth = hs.get_auth()
+
+    async def on_GET(self, request, user_id):
```

Comment:
We've been really bad at asking for this, but can you add type hints here please? The idea is really that new code should be typed where possible

```suggestion
    async def on_GET(self, request: SynapseRequest, user_id: str) -> Tuple[int, JsonDict]:
```

Or something, you'll need to import those types.